app/models/course.rb

```diff
@@ -10,6 +11,8 @@ class Course < ApplicationRecord
   after_initialize :set_defaults, if: :new_record?
   before_validation :set_defaults, if: :new_record?
 
+  validates_with TimeZoneValidator
```

Comment:
Instead of declaring like that, shall we include `validates_with TimeZoneValidator` within `TimeZoneConcern`? The reason for the different validator is because we modified the accessor, which is done in the concern, so every time the concern is included, we need to declare this. If that is the case, might as well include them together. So far I cannot think of a case where we need the concern without the Validator.

Reply:
I can't include `validates_with` inside the concern.

`<module:TimeZoneConcern>': undefined method `validates_with' for TimeZoneConcern:Module (NoMethodError)`

Looks like it might be the way Rails is designed.
https://github.com/rails/rails/issues/11944

Reply:
Ok got it to work, it looks quite nasty but having to remember to include just 1 thing is better.